addins/Cake.Storm.Android.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Storm.Android
```

Comment:
I have always had the impression that these Cake.Storm packages are not really intended for public consumption, and they are built specifically for internal use.  I would suggest we don't add any yml files for Cake.Storm.*

Reply:
I can add them to the "blacklist" which means they will be completely excluded from analysis (not just from yaml generation).